Gems/Atom/Component/DebugCamera/Code/Source/CameraComponent.cpp

```diff
@@ -305,15 +388,42 @@ namespace AZ
 
         void CameraComponent::UpdateViewToClipMatrix()
         {
-            // Note: This is projection assumes a setup for reversed depth
+            // O3de assumes a setup for reversed depth
+            bool reverseDepth = true;
+            
             AZ::Matrix4x4 viewToClipMatrix;
-            MakePerspectiveFovMatrixRH(viewToClipMatrix,
+            MakePerspectiveFovMatrixRH(
+                viewToClipMatrix,
                 m_componentConfig.m_fovY,
                 m_aspectRatio,
                 m_componentConfig.m_depthNear,
-                m_componentConfig.m_depthFar, true);
+                m_componentConfig.m_depthFar,
+                reverseDepth);
+            m_cameraViews[static_cast<uint32_t>(RPI::ViewType::Default)]->SetViewToClipMatrix(viewToClipMatrix);
 
-            m_view->SetViewToClipMatrix(viewToClipMatrix);
+            //Update stereoscopic projection matrix
+            if (m_xrSystem)
+            {
+                AZ::Matrix4x4 projection = AZ::Matrix4x4::CreateIdentity();
+                for (AZ::u32 i = 0; i < m_numXrViews; i++)
+                {
+                    uint32_t xrViewIndex = i == 0 ? static_cast<uint32_t>(RPI::ViewType::XrLeft) : static_cast<uint32_t>(RPI::ViewType::XrRight);
+                    AZ::RPI::FovData fovData;
+                    AZ::RPI::PoseData poseData;
+                    [[maybe_unused]] AZ::RHI::ResultCode resultCode = m_xrSystem->GetViewFov(i, fovData);
+                    resultCode = m_xrSystem->GetViewPose(i, poseData);
```

Comment:
This is getting the view pose from xr system but not using it afterward. Is it necessary?